src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertySymbolBase.cs

```diff
@@ -194,11 +194,11 @@ private enum Flags : byte
 
             if (hasGetAccessor)
             {
-                _getMethod = CreateGetAccessorSymbol(isAutoPropertyAccessor: isAutoProperty, diagnostics);
```

Comment:
>isAutoPropertyAccessor: isAutoProperty,  [](http://example.com/codeflow?start=53&length=40)

I think we should keep passing ```isAutoProperty``` value, but its corresponding parameter should change its name and its meaning. The "bodyShouldBeSynthesizedForSemicolonOnly" name feels like a good fit.